src/test/groovy/com/stormpath/tck/authentication/BearerIT.groovy

```diff
@@ -0,0 +1,23 @@
+package com.stormpath.tck.authentication
+
+import com.stormpath.tck.AbstractIT
+import org.testng.annotations.Test
+
+import static com.jayway.restassured.RestAssured.given
+import static com.stormpath.tck.util.FrameworkConstants.MeRoute
+
+class BearerIT extends AbstractIT {
+
+    @Test(groups=["v100", "json", "html"])
+    public void refreshTokenAsAccessBearerTokenFails() throws Exception {
```

Comment:
Yes. I removed it from that method. It didn't seem right to have a Bearer test in a test meant to exercise cookie operations.